build_integration_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+cli="./faas-cli"
+if [ $(uname) == 'Darwin' ] ; then
+    cli="./faas-cli-darwin"
+elif [ -f "/etc/debian_version" ] ; then
+    if [ $(dpkg --print-architecture) == 'armhf' ] ; then
+        cli="./faas-cli-armhf"
+    elif [ $(dpkg --print-architecture) == 'arm64' ] ; then
+        cli="./faas-cli-arm64"
+    fi
+fi
+
+build_faas_function() {
+
+    function_name=$1
+
+    eval $cli new $function_name --lang python3
+
+cat << EOF > $function_name/handler.py
+def handle(req):
+
+    return "Function output from integration testing: Hello World!"
+EOF
+
+    eval $cli build -f $function_name.yml
+}
+
+run_and_test() {
+    function_name=$1
+    port=$2
+
+    id=$(docker run --env fprocess="python3 index.py" --name $function_name -p $port:8080 -d $function_name:latest)
+
+    curl -s 127.0.0.1:$port > got.txt
```

Comment:
is the service guaranteed to be up at this point, what about on a slow machine like travis. I've seen in other CI openfaas scripts, error handling and waiting for services to start see: https://github.com/openfaas/faas/blob/master/contrib/ci.sh#L11

what do you think?

Reply:
Good point.. some kind of retry would be ideal here.

Reply:
You can curl to `/_/health` which will indicate `200` when ready.